### DISCUSSION TITLE
i18n-ally support

### DISCUSSION THREAD

**Discussion by multipliedtwice** [ivanhofer/typesafe-i18n#98] [DX improvements]
Hello, thank you for your library.
Does it work together with `i18n-ally`?
[https://github.com/lokalise/i18n-ally](https://github.com/lokalise/i18n-ally)

**Comment by ivanhofer** [comment]
Hi @thousandsofraccoons;
I really wish to enable `i18n-ally` support. As of right now there are more important topics on my agenda, so I can't really dive deep into that topic. With a little research i found a [wiki-entry](https://github.com/lokalise/i18n-ally/wiki/Custom-Framework) that describes how to configure it with custom i18n-framworks.
Maybe you want to try it out and comment your findings.
That would help me a lot.
Thanks!

**Comment by ivanhofer** [comment]
for later:
actually it doesn't really look that much of an effort to add official support for a new framework:
It basically means writing a config and opening a PR on the `i18n-ally` repo.
See the examples here: https://github.com/lokalise/i18n-ally/tree/main/src/frameworks

**Comment by ivanhofer** [comment]
I have opened an issue https://github.com/lokalise/i18n-ally/issues/678

**Reply by multipliedtwice** [reply]
@ivanhofer outstanding, thank you.

**Comment by ivanhofer** [comment]
Here is a first working version: https://github.com/lokalise/i18n-ally/issues/678#issuecomment-947338325
